metadata.yml

```diff
@@ -0,0 +1,31 @@
+rules:
+- apiGroups: [""]
+  apiVersions: ["v1"]
+  resources: ["persistentvolumeclaims"]
+  operations: ["CREATE"]
```

Comment:
We track only `CREATE` because `UPDATE` of an existing PVC is not allowed. Am I right?

Reply:
Yeah, as far as I know, `storageClassName` field is immutable. Therefore, there is no need to keep track of update events.